src/__tests__/components/EdgeTextField.test.js

```diff
@@ -0,0 +1,23 @@
+/* globals describe it expect */
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import * as React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+
+import { getTheme } from '../../components/services/ThemeContext.js'
+import { EdgeTextFieldComponent as Request } from '../../components/themed/EdgeTextField.js'
+
+describe('Request', () => {
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      marginRem: 11 | [11],
```

Comment:
This should not have the 'or' sign, it was only part of the type.
You should pass just the number 11.